includes/User/Preferences.php

```diff
@@ -112,9 +112,16 @@ public function register() {
 	/**
 	 * Auth callback.
 	 *
+	 * @since 1.4.0
+	 *
+	 * @param bool   $allowed Unused.
+	 * @param string $meta_key Unused.
+	 * @param int    $user_id User id of user meta.
+	 * @param int    $current_user_id Current user id.
```

Comment:
just some minor docs suggestions.

```suggestion
	 * @param bool   $allowed Unused. Whether the user can add the object meta.
	 * @param string $meta_key Unused. The meta key.
	 * @param int    $user_id ID of the user being edited.
	 * @param int    $current_user_id The currently editing user's ID.
```

I usually tend to keep the original variable names (e.g. `$object_id` & `$user_id`) but I suppose this makes it more understandable

----

Note: need to fix docblock alignment either way.

Reply:
> I usually tend to keep the original variable names (e.g. $object_id & $user_id) but I suppose this makes it more understandable

Me too, it felt wrong here....

Reply:
Fixed.